README.md

```diff
@@ -1 +1,42 @@
-# Unity-Core
\ No newline at end of file
+# Unity-Core
+
+**Unity Core** is an open source Unity plugin that contains all the core functionality for the Ready Player Me Unity SDK and manages all the plugin updates and dependencies including Ready Player Me Avatar Loader and glTFast. 
+
+Please visit the online documentation and join our public `discord` community.
+
+![](https://i.imgur.com/zGamwPM.png) **[Online Documentation]( https://readyplayer.me/docs )**
+
+![](https://i.imgur.com/FgbNsPN.png) **[Discord Channel]( https://discord.gg/9veRUu2 )**
+
+
+## Quick Start 
+***NOTE: This plugin supports Unity Version 2020.3 or higher***
```

Comment:
Could you also note that they need git installed? And drop the URL of it https://git-scm.com

Reply:
ah yes forgot about that 🗡️

Reply:
I added a small requirements section to the start of he quick start 
![image](https://user-images.githubusercontent.com/7085672/207039256-987113e8-54dc-461c-8c8a-08f010ab0bc0.png)